spyderlib/config.py

```diff
@@ -530,6 +557,7 @@ def is_ubuntu():
               'editor/open file': "Ctrl+O",
               'editor/save file': "Ctrl+S",
               'editor/save all': "Ctrl+Shift+S",
+              'editor/save as': 'Meta+S',

```

Comment:
Let's select a different shortcut for *Save as*, given that this one conflicts with Unity and it's kind of useful.

What about `Ctrl+Alt+S`? It seems it's not taken so far.